shared/css/font.scss

```diff
@@ -129,12 +129,16 @@ $barlowSemiCondensed-medium: 'Barlow Semi Condensed Medium', $noto-sans-fonts, s
 $code-font: 'Source Code Pro', Monaco, 'Bitstream Vera Sans Mono', 'Lucida Console', Terminal, monospace;
 
 /*
+   Font Awesome S3 Bucket URLs
+   Current version: 6.6.0
    If you are changing the location of these files,
    see the readme in this directory for more detail on manual steps to take when uploading new files.
  */
-$font-awesome-core-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/fontawesome.min.css';
-$font-awesome-brands-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/brands.min.css';
-$font-awesome-solid-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/solid.min.css';
-$font-awesome-regular-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/regular.min.css';
-$font-awesome-v4-fonts-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/v4-font-face.min.css';
-$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/v4-shims.min.css';
+$font-awesome-core-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/fontawesome.min.css';
+$font-awesome-brands-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/brands.min.css';
+$font-awesome-solid-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/solid.min.css';
+$font-awesome-regular-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/regular.min.css';
+$font-awesome-v4-fonts-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-font-face.min.css';
+$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-shims.min.css';
+$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-shims.min.css';
+$font-awesome-custom-icons-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/custom-icons.min.css';
```

Comment:
should I update the [duotone url](https://github.com/code-dot-org/code-dot-org/pull/61213/files#diff-7011a1f5c1ddf7f7d7ea73ad715c8c10d134fae6245a7bf407a6ad223b9ffab7R141) to use this new version number? I can also merge it in now and you can update it as part of this pr (either way I think we'll get some easily-resolvable merge conflicts)

Reply:
I merged my pr in but can help resolve conflicts/do a follow-up to update the version!

Reply:
No worries, I've just pulled staging and resolved merge conflicts! ✅ 
Re updating url - I've updated in now, just for the future context - different number here stands for different timestamps and versions of font awesome files